src/extension.ts

```diff
@@ -209,6 +209,8 @@ export async function activate(context: vscode.ExtensionContext) {
           email: session.account.label,
           text: sortedPRs[0].title || parsedMessage,
         });
+        */
```

Comment:
This closes, but where does it open?

Reply:
On line 206

Reply:
I will approve but I do not take responsibility for this possible bug